hibernate-core/src/main/java/org/hibernate/boot/registry/BootstrapServiceRegistryBuilder.java

```diff
@@ -24,10 +24,17 @@
 import org.hibernate.service.ServiceRegistry;
 
 /**
- * Builder for {@link BootstrapServiceRegistry} instances.  Provides registry for services needed for
- * most operations.  This includes {@link Integrator} handling and ClassLoader handling.
- *
- * Additionally responsible for building and managing the {@link org.hibernate.boot.registry.selector.spi.StrategySelector}
+ * Builder for {@link BootstrapServiceRegistry} instances.
+ * <p>
+ * An instance of this class may be obtained by instantiations, simply by
```

Comment:
s/instantiations/instantiation

Reply:
In fact I'd reword this sentence 

`An instance of this class may be obtained by {@linkplain #BootstrapServiceRegistryBuilder instantiation}.`

or 

`An instance of this class may be obtained by instantiation; see  {@link #BootstrapServiceRegistryBuilder}.`